aws-s3-transfer-manager/src/operation/download.rs

```diff
@@ -9,10 +9,10 @@ use aws_sdk_s3::error::DisplayErrorContext;
 /// Request type for dowloading a single object from Amazon S3
 pub use input::{DownloadInput, DownloadInputBuilder};
 
-/// Abstractions for response bodies and consuming data streams.
-pub mod body;
 /// Operation builders
 pub mod builders;
+/// Abstractions for responses and consuming data streams.
+pub mod output;
```

Comment:
**fix**: Be consistent across operations. e.g. [upload output module is not public](https://github.com/awslabs/aws-s3-transfer-manager-rs/blob/main/aws-s3-transfer-manager/src/operation/upload.rs#L9) but it [re-exports the output type](https://github.com/awslabs/aws-s3-transfer-manager-rs/blob/main/aws-s3-transfer-manager/src/operation/upload.rs#L23)

Reply:
Thanks, updated.